README.md

```diff
@@ -94,6 +110,9 @@ Your Answer:
 What are the differences between calling and referencing a function? Please provide examples of each.
 
 ```text
+In call by value, you give the function a value, and the function stores that value somewhere locally and works on it.
```

Comment:
[Not exactly!](https://github.com/ga-wdi-quizzes/week-02/blob/solution/README.md#question-8)